src/components/ui/multi-select.tsx

```
@@ -34,202 +39,235 @@ interface MultiSelectProps extends React.ButtonHTMLAttributes<HTMLButtonElement>
     return (
       <div className="w-full">
         <Drawer open={open} onOpenChange={setOpen}>
-          <DrawerTrigger asChild>{triggerButton}</DrawerTrigger>
-          <DrawerContent className="px-0 pt-2">
-            <div className="mt-3 pb-[env(safe-area-inset-bottom)]">
-              {listContent}
+          <DrawerTrigger asChild>
+            <Button
+              variant="outline"
+              ref={ref}
+              role="combobox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)